charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -32,7 +32,7 @@ sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
 # in case of changes within CRDs, a major version bump is mandatory. See: https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#upgrading-chart
-version: 68.3.3
+version: 69.0.0
```

Comment:
Major is okay, but please add a notice in UPGRADING.md

Reply:
will do